cmds/core/dd/dd.go

```diff
@@ -63,21 +62,24 @@ var (
 	bytesWritten int64 // access atomically, must be global for correct alignedness
 )
 
-const (
-	convNoTrunc = 1 << iota
-	oFlagSync
-	oFlagDSync
-)
+type setclear struct {
```

Comment:
`setclear` is not a descriptive name; in fact I thought it meant it'd always clear. flagSetter?

Reply:
bitClearAndSet

maybe flagSetter is better but ... clear happens too ... oh well.